ratatui-book-tutorial-project/src/ui.rs

```diff
@@ -44,12 +44,12 @@ pub fn ui<B: Backend>(f: &mut Frame<B>, app: &App) {
     // ANCHOR: key_value_list
     let mut list_items = Vec::<ListItem>::new();
 
-    for i in 0..app.pairs.len() {
+    for i in app.pairs.keys() {
```

Comment:
nit: `key` would be better than `i` here.

Reply:
I agree, and have made the change